frontend/src/components/CompanyInput.tsx

```diff
@@ -40,18 +40,18 @@ const CompanyInput: React.FC<CompanyInputProps> = ({ onCompanyNameSubmit, onComp
     <div id="root">
       <WelcomeHeader />
       <div className="company-input-container">
-        <h1>Welcome to JobInsights.com</h1>
-        <p>To begin, please enter your company name and click on submit.</p>
+        <h1>Benvenuti su JobInsights.com</h1>
```

Comment:
I would change this section to be in singular form, so
````
Benvenuto su JobInsights.com
Per iniziare, inserisci il nome della tua azienda e clicca su Invia
````

Reply:
fixed this